src/lib/fragments/AgGrid.react.js

```diff
@@ -386,9 +408,20 @@ export default class DashAgGrid extends Component {
     }
 
     componentDidMount() {
+        const {id} = this.props;
+        if (id) {
+            agGridRefs[id] = this.reference.current;
```

Comment:
What if someone changes the ID? I'm not sure if the renderer would actually do this... might always unmount and create a new component. But in case it did, we should probably have `componentDidUpdate` check for a changed ID, delete the old ref and add the new one.